app/src/main/java/com/github/se/polyfit/model/post/PostLocationModel.kt

```diff
@@ -0,0 +1,63 @@
+package com.github.se.polyfit.model.post
+
+import android.Manifest
+import android.content.Context
+import android.content.pm.PackageManager
+import android.location.Location
+import android.util.Log
+import android.widget.Toast
+import androidx.core.app.ActivityCompat
+import com.google.android.gms.location.CurrentLocationRequest
+import com.google.android.gms.location.FusedLocationProviderClient
+import com.google.android.gms.location.LocationServices
+import com.google.android.gms.location.Priority
+import kotlinx.coroutines.tasks.await
+
+class PostLocationModel(private val context: Context) {
+
+  private val currentLocationRequest =
+      CurrentLocationRequest.Builder().setPriority(Priority.PRIORITY_HIGH_ACCURACY).build()
+
+  suspend fun getCurrentLocation(): com.github.se.polyfit.model.post.Location {
+    return checkLocationPermissions()
+  }
```

Comment:
You can import Location and directly use instead of calling com.github...
or an alias if another already got this name

Reply:
The problem: we have a `Location` and what it returns is a `Location` too, so I got some error because the compiler didn't know what Location I was meaning. But it should exist a real solution to this problem so if you know it, i'm all ear

Reply:
With android studio you can import by alias. So the name will be for example location2. So just import Location as location2.
import com.sorandom.app.model.api.Input as OtherInputfun map(input: Input): OtherInput {
  return OtherInput(
    ...
  )
}
https://medium.com/@erickpranata/kotlin-import-alias-3f2831295902